api/mongo_helper.py

```diff
@@ -1,71 +1,139 @@
-        mongo_card_lists = self.lists.find()
-        return [CardList.mongo_dict_to_card_list(mongo_list) for mongo_list in mongo_card_lists]
+        self.logger.debug('Getting lists...')
+        try:
+            mongo_card_lists = self.lists.find()
+            return [CardList.mongo_dict_to_card_list(mongo_list) for mongo_list in mongo_card_lists]
+        except Exception as e:
+            self.logger.error('Error getting lists. Error message: %s', e)
 
     def get_list(self, name):
-        mongo_card_list = self.lists.find_one({'name': name})
-        return CardList.mongo_dict_to_card_list(mongo_card_list)
+        self.logger.debug('Getting list with name: %s ...', name)
+        try:
+            mongo_card_list = self.lists.find_one({'name': name})
+            return CardList.mongo_dict_to_card_list(mongo_card_list)
+        except Exception as e:
+            self.logger.error('Error getting list with name: %s. Error message: %s', name, e)
 
     def move_card_to_list(self, card_id, list_id):
-        datetime_now = datetime.now()
-        self.cards.update_one({'_id': ObjectId(card_id)},{'$set':{'list_id': list_id, 'date_last_activity': datetime_now}})
+        self.logger.debug('Moving Card %s to List %s ...', card_id, list_id)
+        try:
+            datetime_now = datetime.now()
+            self.cards.update_one({'_id': ObjectId(card_id)},{'$set':{'list_id': list_id, 'date_last_activity': datetime_now}})
+        except Exception as e:
+            self.logger.error('Error moving Card %s to List %s. Error message: %s', card_id, list_id, e)
 
     def add_test_db(self, name: str):
-        new_db = self.client[name]
+        self.logger.debug('Creating Test DB with name: %s ...', name)
+        try:
+            new_db = self.client[name]
+        except Exception as e:
+            self.logger.error('Error creating Test DB with name: %s. Error message: %s', name, e)
+        
 
     def drop_test_db(self, name: str):
-        self.client.drop_database(name)
+        self.logger.debug('Dropping Test DB with name: %s ...', name)
+        try:
+            self.client.drop_database(name)
+        except Exception as e:
+            self.logger.error('Error creating Test DB with name: %s. Error message: %s', name, e)
 
-    def _increment_id_short_counter(self):
-        metadata = self.board_metadata.find_one()
-        latest_short_id = metadata['id_short_latest_used']
-        new_short_id = latest_short_id + 1
-        self.board_metadata.update_one({'_id': metadata['_id']},{'$set':{'id_short_latest_used': new_short_id}})
-        return str(new_short_id)
+    def _get_mongo_client(self, mongo_connection_string) -> MongoClient:
+        try:
+            self.logger.debug('Connecting to Mongo...')
+            client = pymongo.MongoClient(mongo_connection_string)
+        except pymongo.errors.ConnectionFailure as e:
+            self.logger.critical('Cannot connect to Mongo. Error message: %s', e)
+            pass
```

Comment:
Now that you've added the logging statement, this `pass` shouldn't be doing anything so we can remove these